tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/ppi_branching_test.py

```diff
@@ -98,7 +99,9 @@ class Observation(object):
     Helper class to initialize test observation rows
     """
 
-    SCHEMA = bq.get_table_schema('observation')
+    project_id = app_identity.get_application_id()
+    bq_client = BigQueryClient(project_id)
+    SCHEMA = bq_client.get_table_schema('observation')
```

Comment:
This feels odd (code smell?) because `get_table_schema` is a static/non-gcloud function but now a BQ client needs to be instantiated to run it. Not a big deal, so this is fine for now.